app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryLabels.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { labelSplit } from '../../../rules/story';
+
+const StoryLabel = ({ label }) => (
+  <a href="#" className="Story__label" title={label}>{label}</a>
+);
+
+StoryLabel.propTypes = {
```

Comment:
Any standard for describing `propTypes` before or after the component definition?

Reply:
oops, didn't notice this is for the component above -- ignore, please.